src/test/java/com/example/rets_api/converter/PropertyConverterTest.java

```diff
@@ -61,6 +63,7 @@ public void entityToDTO_RoomConverter_ShouldReturnCompleteObject(){
         checkAllFields_Room(propertyEntity.getRoomList().get(0), propertyDTO.getRoomList().get(0));
         checkAllFields_School(propertyEntity.getSchoolList().get(0), propertyDTO.getSchoolList().get(0));
         checkAllFields_FinancialData(propertyEntity.getFinancialData(), propertyDTO.getFinancialData());
+//        checkAllFields_AnimalPolicy(propertyEntity.getAnimalPolicyEntity(), propertyDTO.getAnimalPolicy());
```

Comment:
Need to check fields for AnimalPolicy also. Can you please rename the methods from RoomConverter to PropertyConverter, I forgot to fix that :)